### PR TITLE
Attributes separated from entities

### PR DESCRIPTION
**Pull request by ustims**

### CODE REVIEW

File: .idea/.idea.CloudFabric.EAV/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Default ignored files
```
Comment (maxxsem) [inline .idea/.idea.CloudFabric.EAV/.idea/.gitignore:1]: Should we add .idea folder to .gitignore?

File: CloudFabric.EAV.Domain/Models/Attributes/EntityReferenceAttributeConfiguration.cs [hunk 2]
```diff
@@ -13,5 +11,30 @@ public class EntityReferenceAttributeConfiguration : AttributeConfiguration
         public Guid ReferenceEntityConfiguration { get; set; }
```
Comment (maxxsem) [inline CloudFabric.EAV.Domain/Models/Attributes/EntityReferenceAttributeConfiguration.cs:11]: What do you think about using ReferenceEntityConfiguration**Id** just for clarity?

File: CloudFabric.EAV.Domain/Models/Attributes/EntityReferenceAttributeConfiguration.cs [hunk 3]
```diff
@@ -13,5 +11,30 @@ public class EntityReferenceAttributeConfiguration : AttributeConfiguration
...
 
         public Guid DefaultValue { get; set; }
+        
+        public EntityReferenceAttributeConfiguration(
```
Comment (maxxsem) [inline CloudFabric.EAV.Domain/Models/Attributes/EntityReferenceAttributeConfiguration.cs:15]: Do we need a constructor _EntityReferenceAttributeConfiguration(IEnumerable<IEvent> events) : base(events)_ here?
For other attributes as well

**Review by dgon-jd — APPROVED** [review]

File: CloudFabric.EAV.Domain/Models/Attributes/EntityReferenceAttributeConfiguration.cs (hunk 3 above)
Comment (ustims) [reply CloudFabric.EAV.Domain/Models/Attributes/EntityReferenceAttributeConfiguration.cs:15]: Yes, that's the way repository constructs them

**Pull request merged**